binsync/decompilers/ghidra/server/controller.py

```diff
@@ -126,5 +126,15 @@ def functions(self, **kwargs) -> dict:
             return None
         funcs = {}
         for addr in ret:
-            funcs[addr] = Function(addr, ret[addr]["size"], header=FunctionHeader(ret[addr]["name"], addr))        
+            stack_vars = {}
+            for offset in ret[addr]["stack_vars"]:
+                sv = StackVariable(None, None, None, None, None)
+                stack_vars[offset] = sv.__setstate__(ret[addr]["stack_vars"][offset])
+            funcs[addr] = Function(addr, ret[addr]["size"], header=FunctionHeader(ret[addr]["name"], addr), stack_vars=stack_vars)
         return funcs
+
+    def stack_vars(self, addr, **kwargs) -> dict:
```

Comment:
This function does not make much sense. You are getting the stack vars for an address, but then not using that address below? Either use it, or changes the `addr` to `*args`

Reply:
Yeah, I forgot to pass it to ghidra.get_stack_vars()